src/1week/06_송상현.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Action 은 스네이크 표기법으로 작성 (ex. update_tax_dom)
+ * Calc, Data 는 카멜 표기법으로 작성 (ex. addItem)
+ */
+
+// 전역변수와 관련 함수들. 모두 Action
+let shopping_cart = [];
+const fetch_shopping_cart = () => shopping_cart;
+const update_shopping_cart = (cart) => {
+  shopping_cart = cart;
+};
+
+// dom을 가져오거나 변경하는 Action 함수들
+function get_buy_button_dom() {}
+const update_tax_dom = (tax) => console.log(`tax_dom update to ${tax}`);
```

Comment:
```suggestion
const update_tax_dom = tax => console.log(`tax_dom update to ${tax}`);
```
화살표에서 param가 하나일 땐 괄호를 빼는게 깔끔한 것 같습니다! lint에도 그런 규칙이 있더라구요